qtfred/src/ui/dialogs/ShipEditor/ShipFlagsDialog.h

```diff
@@ -17,12 +17,14 @@ class ShipFlagsDialog : public QDialog {
   public:
 	explicit ShipFlagsDialog(QWidget* parent, EditorViewport* viewport);
 	~ShipFlagsDialog() override;
+	void accept() override;
+	void reject() override;
 
   protected:
 	void closeEvent(QCloseEvent*) override;
-
-	void rejectHandler();
   private slots:
+	void on_okButton_clicked();
+	void on_cancelButton_clicked();
```

Comment:
This is fine but these could be replaced with an OK/Cancel button set (QDialogButtonBox) same as other dialogs have used. Then slots then become:

void on_okAndCancelButtons_accepted();
void on_okAndCancelButtons_rejected();

This is how other dialogs have done it so this would just be to match style. This is fine, though, so up to you.

Reply:
Done